addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Alt+GreenFlag 60FPS player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom FPS player mode",
```

Comment:
"Player mode" should be changed to "project player mode" for consistency with the mute addon.